Sources/SwiftHtml/Attributes/Global.swift

```diff
@@ -173,6 +173,7 @@ public extension Tag {
     
     /// Specifies an inline CSS style for an element
     func style(_ value: String) -> Self {
+        guard !value.isEmpty else { return self }
         attribute("style", value)
```

Comment:
You also have to add return before the attribute call in this case. :)

Reply:
Sorry. Finally cloned the GitHub so I can make changes in Xcode with a compiler.

At least I remembered the .isEmpty this time!!!